examples/default/realign.json

```diff
@@ -0,0 +1,67 @@
+{
+  "@context": "https://github.com/Inria-Visages/blob/master/BIDS-prov/context.json",
+  "@id": "http://example.org/ds00000X",
+  "generatedAt": "2020-03-10T10:00:00",
+  "wasGeneratedBy": {
+      "@id": "INRIA",
+      "@type": "Project",
+      "startedAt": "2016-09-01T10:00:00",
+      "wasAssociatedWith": { "@id": "NIH",
+                              "@type": "Organization",
+                              "hadRole": "Funding"
+                            }
+    },
+    "exp_dir" : {
+      "@id" : "exp_dir",
+      "prov:atLocation" : "$HOME/BIDS-prov/nidm-results-examples/spm_default/ds011/"
+    },
+  "@graph": [
+    {
+      "prov:Activity" : [
+        {
+          "@id" : "realign",
+          "wasAssociatedWith" : "estwrite",
+          "startedAtTime" : "10/10/2020 00:00:00",
+          "endedAtTime" : "10/10/2020 01:00:00",
+          "used" : "$HOME/BIDS-prov/nidm-results-examples/spm_default/ds011/PREPROCESSING/ANATOMICAL/*",
+          "attributes" : [
+            ["eoptions.quality", 0.9],
+            ["eoptions.sep", 4],
+            ["roptions.which", [0, 1]],
+            ["roptions.interp", 4]
+          ]
+        },
+        {
+          "@id" : "coreg.estimate",
+          "wasAssociatedWith" : {
+              "@type" : "prov:SoftwareAgent",
+              "RRID" : "https://scicrunch.org/resources/Any/record/nlx_144509-1/SCR_007037/resolver?q=SPM&l=SPM",
+              "label" : "SPM"
+            },
+            "wasInformedBy" : "realign",
```

Comment:
I would remove the "wasInformedBy", the link with "realign" is made through the input/output files.